src/index.js

```diff
@@ -4,6 +4,19 @@ import colorMap from './colorMap';
 const NIL = 0x00;
 
 /**
+ * Check if array contains valid (0-255) input
+ * @param  {array} nums Array of values
+ * @returns {bool} True if any values in array are invalid
+ */
+function invalidNums(nums) {
+  const invalid = nums.find(num => {
+    return !Number.isInteger(num) || num < 0 || num > 255;
+  });
+
+  return invalid;
+}
```

Comment:
Because we're not using the returned value, it might make sense to send back a bool```js
function invalidArguments(nums) {
  return nums.some(num =>
    !Number.isInteger(num) || num < 0 || num > 255;
  );
}
```